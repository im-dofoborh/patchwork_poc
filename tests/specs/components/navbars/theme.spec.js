import { mount, createLocalVue } from '@vue/test-utils';
import Vuex from 'vuex';
import ThemeNavbar from '~/components/navbars/Theme';
import { site } from '~/tests/fixtures';

describe('ThemeNavbar component', () => {
  let localVue = null;
  let wrapper = null;
  let store = null;

  beforeEach(() => {
    localVue = createLocalVue();
    localVue.component('theme-navbar', ThemeNavbar);
    localVue.use(Vuex);
    store = new Vuex.Store({
      state: {
        sites: [
          site
        ],
      },
    });
    wrapper = mount(ThemeNavbar, {
      localVue,
      store,
    });
  })

  it('renders default variant', () => {
    expect(wrapper.html()).toMatchSnapshot();
  });
});
