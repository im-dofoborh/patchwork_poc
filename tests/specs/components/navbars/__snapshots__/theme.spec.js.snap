// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ThemeNavbar component renders default variant 1`] = `
<nav class="sg-theme-navbar"><select autocomplete="off" class="sg-theme-navbar__select">
    <option> -- View in Brand -- </option>
    <option value="slate">Slate</option>
  </select></nav>
`;
